enhancements/machine-api/machine-api-usage-telemetry.md

```diff
@@ -0,0 +1,340 @@
+as initial inquries we would like to gather and report. They have been
+categorized into 3 epics: demonstration of value, engineering insights,
+and proactive support. Demonstration of value speaks to the raw amount of
+usage that is happening for any given component. Engineering insights speaks
+to situations where we can learn about user experience and interactions to inform
+future planning and guidance. Proactive support speaks to information that we
+learn which will lead to better experiences for users by providing opportunity
+for preventitive intervention and education.
+
+**Total Remediation Events**
+
+Epics: Demonstration of Value, Proactive Support
+
+This metric will give us insight into how effectively the Machine Health
+Check is being used. Seeing a sustained high number of events on a particular
+cluster provides opportunities to deliver proactive support. Seeing a sudden
+spike across all clusters might give us early warning to a cloud provider’s data
+center outage, or a large network or similar event that is impacting many of our
+users. Having a total number of remediation events can demonstrate the value
+our platform is bringing to our users. It will show the operational burden
+we’ve decreased on teams that use OpenShift.
+
+For example, since many clusters operate at a relatively small scale, it’s
+possible that losing an Availability Zone in AWS US-East-1 region would be
+small enough event on their cluster to trigger the Machine Health Check. Letting
+users know we’re seeing this across many clusters would assist in their own troubleshooting
+efforts and save them time.
+
+**Remediation is Disabled or Short Circuited**
+
+Epics: Proactive Support
+
+This metric will give us insight into when a particular cluster has too many
+unhealthy nodes for the Machine Health Check to do its job. A user may be expecting
+that the health check is running as normal, cleaning up broken hosts, when in
+reality it has been silently disabled. Any sustained disabling of the Machine
+Health Check should potentially trigger a proactive support engagement. Similar
+to the Total Remediation Events, experiencing a high volume of these events
+across several users or platforms provides opportunities to create alerts for
+users of potential outage events.
+
+**Number of Nodes Covered by the Machine Health Check**
+
+Epics: Engineering Insights, Proactive Support
+
+This metric will give us insight into user experiences operating the Machine Health
+Check. For example, users might think they are utilizing the Machine Health
+Check correctly only to find out that it has been misconfigured and not
+performing its role. If users have a large cluster with the health check
+enabled, but not covering any nodes, this might be an opportunity to reach out
+and see if we can help with their configuration. This can also help us identify
+areas where increased education will have value.
+
+**Failed Node Cleanups**
```

Comment:
It's not clear this story is related to the cluster-autoscaler just from reading.

Reply:
i've updated the first sentence to be more clear, let me know if that works